.gitmodules

```diff
@@ -3,8 +3,7 @@
 	url = https://github.com/qmk/ChibiOS
 [submodule "lib/chibios-contrib"]
 	path = lib/chibios-contrib
-	url = https://github.com/qmk/ChibiOS-Contrib
-	branch = k-type-fix
+	url = https://github.com/drashna/ChibiOS-Contrib
```

Comment:
This should/will be changed back to QMK's fork once qmk/ChibiOS-Contrib#12 has been merged in.